frontend/src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,39 @@
+const DropDowonComponent = (buttonValue: any, values: Array<any>, handleOnClick: Function) => {
```

Comment:
There is a typo...
And we don't need to name react component SomethingComponent (it is expected to be a component)...
And dropdown is one word so there is no need to use case like (dropDown/DropDown)